public/index.html

```diff
@@ -24,7 +24,19 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Shoes Rack</title>
+    <link
```

Comment:
This is not important for now, but you should know that the more fonts you add the slower your page will be, so make sure you don't add too many fonts / weights!